resources/assets/components/ReviewBlock/index.js

```diff
@@ -11,6 +11,7 @@ class ReviewBlock extends React.Component {
 
   render() {
     const post = this.props.post;
+    const disableTags = post.status === 'accepted' || post.status === 'rejected' ? false : true;
```

Comment:
This reads a little awkwardly to me with the ternary... could we reformat like this?

```js
const disableTags = ! (post.status === 'accepted' || post.status === 'rejected');
```